### PR TITLE
Add params

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by gnurag** [comment]
cc @kgaikwad @r14chandra

**Review by kgaikwad — APPROVED** [review]
Looks good to me :+1:

File: deployment.yaml [hunk 1]
```diff
@@ -89,6 +89,12 @@ parameters:
 - description: memory limit of service
   name: MEMORY_LIMIT
   value: 512Mi
+- description: Cpu request of service
```
Comment (kgaikwad) [inline deployment.yaml:92]:
```suggestion
- description: CPU request of service
```
Just one minor suggestion.

**Review by r14chandra — APPROVED** [review]
LGTM

**Comment by r14chandra** [comment]
LGMT

**Pull request merged**